test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14235,7 +14240,10 @@ def create_eatcpu_job(self, duration=None, mom=None):
                                   'bin', 'pbs_python')
         a = {ATTR_S: shell_path}
         self.set_attributes(a)
-        self.du.chmod(path=script_path, mode=0o755)
+        mode = 0o755
+        if not self.du.chmod(path=script_path, mode=mode, sudo=True):
```

Comment:
Does this need a host?

Reply:
Can you please elaborate?  @ericpershey can correct me if I am wrong, but the environment in which we have been testing is a single node running both the server and a mom.  My understanding from the du.chmod docstring is that specifying a node is only necessary in a multi-node environment.  Again, we are new to PBS Pro and my reading of the code and resulting output could be wrong, but our observation is that the eatcpu.py code installed in /opt/ptl is missing the executable bit which results in the test failing when the test is on a single node.

Reply:
@toonen I think he is right, it needs a host.  The signature of create_eatcpu_job does contain mom, a hostname, and that hostname is passed to run_copy.  @toonen and I talked about adding it to run_copy, but some of the other functions seem to do a run_copy then chmod right after.  Thoughts?